src/libs/OptionsListUtils.js

```diff
@@ -92,11 +93,14 @@ function createOption(personalDetailList, report, draftComments, activeReportID,
             : '')
         + _.unescape(report.lastMessageText)
         : '';
+    const tooltipText = getReportParticipantsTitle(lodashGet(report, ['participants'], []));
```

Comment:
```suggestion
    const tooltipText = getReportParticipantsTitle(report.participants || []);
```

Reply:
As per the old implementation the report may be null so Instead of doing `report? report.participants : []`. I did this.